Documentation/Signoffs/Power-Subsystem.md

```diff
@@ -0,0 +1,102 @@
+| 6 | OpAmp Circuit | The opamp circuit requires +/- 5 V. The maximimum input voltage is +/- 15 V, and the maximum power consumption is going to be from 8 to 60 mW | OP07 Datasheet [13] | 
+
+
+## Buildable Schematic
+![Power Subsystem Buildable Schematic](https://github.com/Michaelwwest98/DARPA-Drone-Triage-Sensing-System/assets/123699820/a4a049ab-8726-47c9-a8cd-c67424ba8cbb)
+
+## Analysis
+1. The system must operate at full functionality for 15 to 60 minutes. The NP-F550 7.4 V, 2600 mAh is estimated to power the Jetson Nano with a 4 A draw for approximatley 40 minutes. The radar module, transmission module, microphone, and speaker will be powered directly from the Jetson Nano. 
+2. The form of power shall be a battery. The battery chosen to power the sensing system is a 2 cell lithium-ion battery pack that is rechargable, relativley small in size, and low in weight in order to eventually be mounted to a drone.  
+3.  The power system must be under 1.5 lbs. The power system is weighing in at just under 1 lb. This design complies with the weight constraint [5][6][8].
+4.  The Jetson Nano requires a specific power supply. The buck converter/regulator is going to step down the voltage from 12 V to 5.0 V and limits the current to a maximum of 5 A. The buck converter uses op amps as feedback circuit to regulate output voltage. The battery adapter plate is going to allow easy connection to the battery pack and the regulator. The system will require two 2.1 mm DC barrel jacks with loose leads to connect the battery to the input of the regulator and the second one to connect the output of the regulator to the power supply input of the Jetson Nano. The connectors that will be used are 18 AWG which will be capable of handling 5V/5A. This design makes for an easy setup and meets all specifications for the Jetson Nano [5][6][7].
+
+Input Voltage Ripple considerations:
+| # | Peripheral | Ripple Considerations | Source |
+|---|------------|----------------------|--------|
+| 1 | Radar Module | Max ripple 500 mV peak to peak | IPS-937 24GHZ RADAR MODULE datasheet [9] |
+| 2 | RAK811 LPWAN Breakout Module | Max ripple 220 mV peak to peak | RAK811 LPWAN Breakout Module datasheet [10] |
+| 3 | Microphone | Max ripple 500 mV peak to peak | B0B-19389 datasheet [11] |
+| 4 | Speaker | Max ripple 500 mV peak to peak | COM-18343 datasheet [12] |
+
+- Due to the peripherals being powered off of the Jetson's power rails the maximum input voltage ripple is the lowest of the peripherals which is 220 mV peak to peak. 
+
+![Buck converter output voltage ripple](https://github.com/Michaelwwest98/DARPA-Drone-Triage-Sensing-System/assets/123699820/1b264d8e-653e-47e7-ab21-49f261f8b0f4)
+
+- The simulated buck converter circuit above shows an output voltage ripple of around 200 mVp-p that does not dip below 4.75 V which is within standard for the Jetson Nano and all of the peripherals.
+
+5. 
```

Comment:
You have to prove that the Jetson can do this. It's not enough to say what it needs to do. It must be shown that the Jetson will do it rather than assumed.